packages.ini

```diff
@@ -112,6 +112,9 @@ brew_requires = libffi
 [cffi==1.15.1]
 apt_requires = libffi-dev
 brew_requires = libffi
+[cffi==1.16.0]
+apt_requires = libffi-dev
+brew_requires = libffi
```

Comment:
This is a byproduct of calling `python3 -m add_pkg sentry-relay`. It should not do any harm.